app/assets/javascripts/expand.js

```diff
@@ -50,11 +50,11 @@ $(document).ready(function() {
       delete_button.setAttribute('data-method','delete');
       // delete_button.setAttribute('class','tiny update-butt');
 
-      update_button.appendChild(update_text)
-      delete_button.appendChild(delete_text)
+      update_button.appendChild(update_text);
```

Comment:
Identifier 'update_button' is not in camel case.<br>Identifier 'update_text' is not in camel case.